.github/workflows/vib-verify.yaml

```diff
@@ -93,4 +93,48 @@ jobs:
           # Target-Platform used by default
           VIB_ENV_TARGET_PLATFORM: ${{ secrets.VIB_ENV_TARGET_PLATFORM }}
           # Alternative Target-Platform to be used in case of incompatibilities
-          VIB_ENV_ALTERNATIVE_TARGET_PLATFORM: ${{ secrets.VIB_ENV_ALTERNATIVE_TARGET_PLATFORM }}
\ No newline at end of file
+          VIB_ENV_ALTERNATIVE_TARGET_PLATFORM: ${{ secrets.VIB_ENV_ALTERNATIVE_TARGET_PLATFORM }}
+  ci-pr-review:
```

Comment:
We should have added a `name` here to avoid this:

<img width="853" alt="Screenshot 2022-06-14 at 20 20 06" src="https://user-images.githubusercontent.com/1021745/173661250-91b7179b-6463-47d9-b50c-310676f85426.png">

Reply:
IMO, the jobs **Get modified charts** and **ci-pr-review** don't belong to the VIB service.

Reply:
I do not have any strong thoughts against leaving only the **Verify** job in the VIB workflow. As for my thought process here, I ended up leaving it as part of the same VIB workflow due to the data the **ci-pr-review** job needs from the **Verify** one. 

GitHub doesn't provide an easy solution to pass data between workflows; we'll have to use workflow artifacts and the logic gets more complex. The same would happen if we extract the **Get modified charts**.